client/src/lib/utils/config.ts

```diff
@@ -1,3 +1,5 @@
 export const CAPTCHA_KEY: string = import.meta.env.VITE_CAPTCHA_KEY;
 export const FAUCET_URL = import.meta.env.VITE_FAUCET_URL;
 export const DEMO_MODE: boolean = !!import.meta.env.VITE_DEMO;
+
+export const CHAINS: { name: string, id: string }[] = [{ name: 'Rococo', id: '1002' }];
```

Comment:
the parachain_id should not be hardcoded with a network, as there are many other parachain_ids within Rococo
the 1002 is actual only in scope of INK docs. 
so here it should be a just number input, optional, as it's used rarely in `Rococo Testnet faucet`
![image](https://user-images.githubusercontent.com/1177472/221576845-fd2d28f0-6dab-403a-91c5-0b28e67c43d3.png)
Most often people just drip to their account within rococo chain

Reply:
If we start using a list of chains here, I guess this also requires to refactor the config for faucet URLS, as it should be a map
VITE_FAUCET_URL == rococo,
but how we gonna include westend ?